operator/src/main/java/oracle/kubernetes/operator/TuningParametersImpl.java

```diff
@@ -134,4 +137,28 @@ public PodTuning getPodTuning() {
       lock.readLock().unlock();
     }
   }
+
+  // path - a file containing a base64 encoded string containing the operator's cert in pem format
+  public String getFileContents(String path) {
+    // in pem format
+    String result = null;
+    if (checkFileExists(path)) {
+      try {
+        result = new String(Files.readAllBytes(Paths.get(path)));
+      } catch (Throwable t) {
+      }
```

Comment:
LOGGER.warning(MessageKeys.EXCEPTION, t);

Reply:
Thanks.  Fixed.  I didn't know there was a generic exception message for logging.